Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -34,13 +35,12 @@ public class DefaultConsumerErrorStrategy : IConsumerErrorStrategy
         private readonly object syncLock = new object();
 
         private IConnection connection;
-        private bool errorQueueDeclared;
-        private readonly ConcurrentDictionary<string, string> errorExchanges = new ConcurrentDictionary<string, string>();
+        private readonly ConcurrentDictionary<string, object> existingErrorExchangesWithQueues = new ConcurrentDictionary<string, object>();
```

Comment:
The dictionary of <RoutingKey, ExchangeName> was enough only for default case, where the only variable in exchange names is routing key. As soon as I start introducing custom exchange and queue names composed from original queue names, we need unique identifier for whole error topology.
I've used this convention for caching create topologies = errorTopologyIdentifier = $"{errorExchangeName}-{errorQueueName}-{routingKey}"; see code below